labshare/templatetags/icon.py

```diff
@@ -0,0 +1,29 @@
+from django import template
+
+register = template.Library()
+
+def extract_string(value):
+  if not (value and value[0] == value[-1] and value[0] in ('"', "'")):
+    return value
+  return value[1:-1]
+
+class IconNode(template.Node):
+  def __init__(self, icon_name):
+    self.icon_name = icon_name
+
+  def render(self, context):
+    return "<span class=\"glyphicon glyphicon-%s\"></span> " %(self.icon_name)
```

Comment:
could you use a 'modern' format string like: `"<span class=\"glyphicon glyphicon-{icon}\"></span> ".format(icon=self.icon_name)`?